### PR TITLE
Take the width of block's prefix into account only once 

### LINKED ISSUE #2289: line length calculation off by 7 characters for unsafe block
With rustfmt-nightly 0.3.2, the formatting of the below code seems to have a miscalculation of 7. The `identifier_1` code is only one character longer than the `identifier_0`code, and should be formatted in one line as well. And the `identifier_3` code is only one character longer than the `identifier_2` code, and should be formatted in two lines as well.

```rust
fn foo() {
    let identifier_0 = unsafe { this_is_58_chars_long_and_line_is_93_chars_long_xxxxxxxxxx };
    let identifier_1 =
        unsafe { this_is_59_chars_long_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx };
    let identifier_2 =
        unsafe { this_is_73_chars_long_and_line_is_93_chars_long_xxxxxxxxxxxxxxxxxxxxxxxxx };
    let identifier_3 = unsafe {
        this_is_74_chars_long_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
    };
}
```

### PR DESCRIPTION
**Pull request by topecongiro**
Closes #2289.

### CODE REVIEW

File: tests/source/expr.rs [hunk 1]
```diff
@@ -108,6 +108,12 @@ fn baz() {
     unsafe {
         foo();
     }
+
+    // #2289
+    let identifier_0 = unsafe { this_is_58_chars_long_and_line_is_93_chars_long_xxxxxxxxxx };
+    let identifier_1 = unsafe { this_is_59_chars_long_and_line_is_94_chars_long_xxxxxxxxxxx };
+    let identifier_2 = unsafe { this_is_65_chars_long_and_line_is_99_chars_long_xxxxxxxxxxxxxxxxx };
+    let identifier_3 = unsafe { this_is_66_chars_long_and_line_is_100_chars_long_xxxxxxxxxxxxxxxxx };
```
Comment (tspiteri) [inline tests/source/expr.rs]: Line lengths for identifier_2 and identifier_3 should be 100 and 101 chars respectively, not 99 and 100.

Comment (topecongiro) [reply tests/source/expr.rs]: Thank you for the review, they are actually 100 and 101 chars respectively, just typo on variable names.

**Comment by topecongiro** [comment]
Fix typos on variable names.

**Comment by nrc** [comment]
Thank you!

**Pull request merged**